src/status_im/multiaccounts/recover/core.cljs

```diff
@@ -160,15 +158,21 @@
    (navigation/navigate-to-cofx :recover-multiaccount-enter-phrase nil)))
 
 (fx/defn proceed-to-import-mnemonic
-  {:events [:multiaccounts.recover/enter-phrase-next-pressed]}
-  [{:keys [db] :as cofx}]
+  {:events [:multiaccounts.recover/phrase-validated]}
+  [{:keys [db] :as cofx} phrase-warnings]
   (let [{:keys [password passphrase]} (:intro-wizard db)]
-    (if (check-phrase-warnings passphrase)
+    (if-not (= (get-in (types/json->clj phrase-warnings) [:error]) "")
       (popover/show-popover cofx {:view :custom-seed-phrase})
       (when (mnemonic/valid-length? passphrase)
         {::import-multiaccount {:passphrase passphrase
                                 :password   password}}))))
 
+(fx/defn seed-phrase-next-pressed
+  {:events [:multiaccounts.recover/enter-phrase-next-pressed]}
+  [{:keys [db] :as cofx}]
+  (let [{:keys [passphrase]} (:intro-wizard db)]
+    (status/validate-mnemonic-async passphrase #(re-frame/dispatch [:multiaccounts.recover/phrase-validated %]))))
```

Comment:
you need to create fx function and return here `{:validate-mnemonic [passphrase callback]}`

Reply:
Do you mean just turn `validate-mmemonic-async` into an fx/defn in `native-module` and then dispatch it here?

Reply:
you need to reg fx here 
```
(re-frame/reg-fx
 :validate-mnemonic-async
 (fn [[passphrase callback]]
   (status/validate-mnemonic-async passphrase callback)))
```

and return this map
```
(let [{:keys [passphrase]} (:intro-wizard db)]
  { :validate-mnemonic-async [passphrase #(re-frame/dispatch [:multiaccou....]]}
```

Reply:
Thanks!  Made the corrections in my latest commit.